tests/xybeamstop.py

```diff
@@ -0,0 +1,164 @@
+        self.ca.assert_that_pv_is_close(MTR1, math.pi/2.)
+        self.ca.assert_that_pv_is_close(MTR2, 0)
+
+    def test_GIVEN_store_command_THEN_cannot_move_arm(self):
+        # store the arm
+        self._set_pv_value("ARM:STORE:SP", 1)
+
+        # Now try and move it. This should not move as the motor has put access disabled.
+        self._set_x(0.2)
+        self._set_y(0.2)
+
+        time.sleep(1)
+
+        self.ca.assert_that_pv_is("MOT:ARM:STORE", "STORED")
+        self.ca.assert_that_pv_is_close("MOT:ARM:X", -7.071)
+        self.ca.assert_that_pv_is_close("MOT:ARM:Y", 2.929)
+        self.ca.assert_that_pv_is_close("MOT:ARM:X.RBV", -7.071, 1e-2)
+        self.ca.assert_that_pv_is_close("MOT:ARM:Y.RBV", 2.929, 1e-2)
+        self.ca.assert_that_pv_is_close(MTR1, math.pi/2.)
+        self.ca.assert_that_pv_is_close(MTR2, 0)
+
+    def test_GIVEN_activate_command_sent_in_stored_mode_WHEN_read_x_y_is_as_expected(self):
+        self._set_pv_value("ARM:STORE", 1)  # First put it into stored mode
+        self._set_pv_value("ARM:STORE", 0)  # Then check it comes back out
+
+        time.sleep(2)
+
+        self.ca.assert_that_pv_is("MOT:ARM:STORE", "ACTIVE")
+        self.ca.assert_that_pv_is("MOT:ARM:X", 0.)
+        self.ca.assert_that_pv_is("MOT:ARM:Y", 0.)
+        self.ca.assert_that_pv_is_close("MOT:ARM:X.RBV", 0, 0.6)
+        self.ca.assert_that_pv_is_close("MOT:ARM:Y.RBV", 0, 0.6)
+        self.ca.assert_that_pv_is_close(MTR1, math.pi/4., 1e-3)
+        self.ca.assert_that_pv_is_close(MTR2, 0, 1e-3)
+    #
+    # def test_GIVEN_set_x_outside_lower_limit_WHEN_read_x_is_not_outside_limit(self):
+    #     x_lower = -3.535
+    #     self._set_x(x_lower)
+    #     self._set_x(x_lower - 1.0)
+    #     time.sleep(1)
+    #     self.ca.assert_that_pv_is("MOT:ARM:X", x_lower)
+    #     self.ca.assert_that_pv_is_close("MOT:ARM:X.RBV", x_lower, 1e-1)
+    #
+    # def test_GIVEN_set_x_outside_upper_limit_WHEN_read_x_is_not_outside_limit(self):
+    #     x_upper = 3.535
+    #     self._set_x(x_upper)
+    #     self._set_x(x_upper + 1.0)
+    #     time.sleep(1)
+    #     self.ca.assert_that_pv_is("MOT:ARM:X", x_upper)
+    #     self.ca.assert_that_pv_is_close("MOT:ARM:X.RBV", x_upper, 1e-1)
+    #
+    # def test_GIVEN_set_y_outside_lower_limit_WHEN_read_y_is_not_outside_limit(self):
+    #     y_lower = -7.071
+    #     self._set_y(y_lower)
+    #     self._set_y(y_lower - 1.0)
+    #     time.sleep(1)
+    #     self.ca.assert_that_pv_is("MOT:ARM:Y", y_lower)
+    #     self.ca.assert_that_pv_is_close("MOT:ARM:Y.RBV", y_lower, 1e-1)
+    #
+    # def test_GIVEN_set_y_outside_upper_limit_WHEN_read_y_is_not_outside_limit(self):
+    #     y_upper = 2.929
+    #     self._set_y(y_upper)
+    #     self._set_y(y_upper + 1.0)
+    #     time.sleep(1)
+    #     self.ca.assert_that_pv_is_close("MOT:ARM:Y", y_upper)
+    #     self.ca.assert_that_pv_is_close("MOT:ARM:Y.RBV", y_upper, 1e-2)
```

Comment:
Are these tests not being run for a reason? Please either reactivate them or get rid of them